coderd/coderd.go

```diff
@@ -1002,6 +1002,9 @@ func New(options *Options) *API {
 						})
 					})
 				})
+				r.Route("/paginated-members", func(r chi.Router) {
+					r.Get("/", api.paginatedMembers)
+				})
```

Comment:
```suggestion
				r.Get("/paginated-members", api.paginatedMembers)
```

I wouldn't bother with the subrouter here

Reply:
I saw this was a pattern we were doing in other places so copied that. Will change